### PR TITLE
fixing group/order

### PR DESCRIPTION
**Pull request by rafiki270**
Fixes https://github.com/vapor/sql/issues/2

### CODE REVIEW

File: Tests/SQLTests/DataQueryTests.swift [hunk 1]
```diff
@@ -62,10 +62,11 @@ final class DataQueryTests: XCTestCase {
 
         let column = DataComputedColumn(function: "YEAR", columns: [.init(table: "foo", name: "date")])
         select.groupBys = [.computed(column)]
+        select.orderBys = [DataOrderBy(columns: [DataColumn(table: "foo", name: "name")], direction: .descending)]
```
Comment (tanner0101) [inline Tests/SQLTests/DataQueryTests.swift]: can we add a sort to this test as well to make sure the ordering is correct going forward :)

Comment (rafiki270) [reply Tests/SQLTests/DataQueryTests.swift]: I am confused? you just commented on it @tanner0101? Or would you like a separate test only to check that?

**Comment by penny-coin** [comment]
Hey @rafiki270, you just merged a pull request, have a coin! 

You now have 98 coins.

**Comment by tanner0101** [comment]
thanks!

**Pull request merged**